INSTALL.md

```diff
@@ -91,7 +105,6 @@ api:
   auth_token: AUTH_TOKEN
 runner:
```

Comment:
I think this example should be removed or made more minimal, and the linux-specific stuff moved into the linux install guide (as the mac default config is in the mac guide)